File: app/src/main/java/dev/ragnarok/fenrir/api/model/VKApiStory.java
package dev.ragnarok.fenrir.api.model;

public class VKApiStory implements VKApiAttachment {

    /**
     * Note ID, positive number
     */
    public int id;

    /**
     * Note owner ID.
     */
    public int owner_id;

    /**
     * Date (in Unix time) when the note was created.
     */
    public long date;

    public long expires_at;

    public boolean is_expired;

    public boolean is_ads;

    public String access_key;

    public String target_url;

    public VKApiStory parent_story;

    public VKApiPhoto photo;

    public VKApiVideo video;

    @Override
    public String getType() {
        return TYPE_STORY;
    }
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/VKApiStory.kt
package dev.ragnarok.fenrir.api.model

class VKApiStory : VKApiAttachment {
    /**
     * Note ID, positive number
     */
    var id = 0

    /**
     * Note owner ID.
     */
    var owner_id = 0

    /**
     * Date (in Unix time) when the note was created.
     */
    var date: Long = 0
    var expires_at: Long = 0
    var is_expired = false
    var is_ads = false
    var access_key: String? = null
    var target_url: String? = null
    var parent_story: VKApiStory? = null
    var photo: VKApiPhoto? = null
    var video: VKApiVideo? = null
    override fun getType(): String {
        return VKApiAttachment.TYPE_STORY
    }
}